docs/eventlisteners.md

```diff
@@ -954,3 +959,11 @@ Except as otherwise noted, the content of this page is licensed under the
 [Creative Commons Attribution 4.0 License](https://creativecommons.org/licenses/by/4.0/),
 and code samples are licensed under the
 [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
+
+## EventListener Secure Connection
+
+Triggers now support both `HTTP` and `HTTPS` connection by adding few configurations to eventlistener.
```

Comment:
Could we add a little bit more detail here? e.g. :

even a like like -- To setup TLS, add set two environment variables `TLS_CERT`, and `TLS_KEY` with values set to a Secret. See the full example(LINK) for details`

Reply:
Updated